main.js

```diff
@@ -1,36 +1,35 @@
-"use strict";
+const squirrelStartup = require('electron-squirrel-startup');
+// windows only: don't run app during squirrel-install
+function exit() { return; }
```

Comment:
This is probably quite controversial.. There is one rule concerning the scope of returns. It basically disallows the use of `return` outside of function scopes/in global the scope. 

This means a construct like:
```
if (true) { return };
```
is not valid under this rule, thus that proxy function `exit()`.